### PR TITLE
implement LIKEs for extended attributes table

### PR DESCRIPTION
**Pull request by mtmcgrew**
implement LIKEs for extended attributes table. basically come copy paste from file.cpp

### CODE REVIEW

**Comment by osqueryer** [comment]
:no_good: The commit c94377a296c667892bce34f240129e30a6918b94 (Job results: [1226](https://jenkins.osquery.io/job/osqueryPullRequestAudit/1226/)) failed the code audit or documentation test.

File: osquery/tables/system/darwin/extended_attributes.cpp [hunk 1]
```diff
@@ -227,18 +245,41 @@ QueryData genXattr(QueryContext &context) {
...
+    for (const auto& directory_string : directories) {
+      if (!isReadable(directory_string) || !isDirectory(directory_string)) {
+        continue;
+      }
 
-    for (auto &file : files) {
-      getFileData(results, file);
+      try {
+        // Iterate over the directory and generate info for each regular file.
+        fs::directory_iterator begin(directory_string), end;
```
Comment (theopolis) [inline osquery/tables/system/darwin/extended_attributes.cpp]: Will this recurse into the directory?

**Comment by theopolis** [comment]
Do a `make format_master` and then push the inline changes made by `clang-format`.

**Comment by theopolis** [comment]
Ping!

**Comment by mtmcgrew** [comment]
Sorry have not had time for this, will try to get to today.

**Comment by jedi22** [comment]
Ping!

**Pull request closed without merging**